### DISCUSSION TITLE
Main Window

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#297] [Ideas] [answered]
A brother gave an idea. Would it be possible to further reduce the size of the JWMMF window, and make it adapt to what is in it? As a reference, he cited the OnlyM program.

**Comment by sircharlo** [comment, marked as answer]
The window is actually resizable!
Grab any one of the sides to give it a whirl. 

What do you mean by adapt?

**Reply by joaochris** [reply]
The main window has a lock at a certain size right? When you get to the minimum, you can't adjust. 

What do you mean by adapt?
Adapt program depending on the size of the window, either minimum or maximum to text, icons, etc.

**Reply by sircharlo** [reply]
Do you mean the main window or the media presentation window?
I suppose I could "unlock" the main window; out of curiosity, in what possible scenario would the window be *that* small? 😂

**Reply by joaochris** [reply]
That's the main window. 😂 I believe it's for taste, and to put more programs on the screen.

**Reply by sircharlo** [reply]
Why not🙂. I'll see what I can do.

**Reply by joaochris** [reply]
My friend also told me this today 😂. Thanks 😎

**Reply by sircharlo** [reply]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/4fa20170e2dd04856e3b19369dd216064047bebc
